app/controllers/warehouse_reports/client_details/last_permanent_zips_controller.rb

```diff
@@ -28,8 +28,9 @@ def index
       end
 
       @clients = population_service_history_source.
-        joins(:client, :enrollment, :project).
-        includes(:client, :enrollment, :project).
+        joins(:client, :enrollment, :project, :organization).
+        includes(:client, :enrollment, :project, :organization).
+        preload(:organization). # FIXME why isnt this working, still a bunch of organization queries
```

Comment:
Trying to get rid of N+1 that's happening [here](https://github.com/greenriver/hmis-warehouse/blob/21aeb8ddaf1fc6d7ff778767d886a417b5eda5b3/app/views/warehouse_reports/client_details/last_permanent_zips/index.haml#L47), but this doesn't seem to fix it. @eanders can we look at this together at some point?